File: represent/bench/c_flat/SpecGenBench_0308__SpecGenBench__common__IsAscending__IsAscending.c
int isAscending(int * arr, int arr_len);

int isAscending(int * arr, int arr_len) {

        int n = arr_len;
        if (n < 2) {
            return 1;
        }
        for (int i = 0; i < n; i++) {
            if(arr[i] >= arr[i+1])
                return 0;
        }
        return 1;
}

File: represent/bench/java_flat/SpecGenBench_0308__SpecGenBench__common__IsAscending__IsAscending.java
public class IsAscending {
    public boolean isAscending (int[] arr) {
        int n = arr.length;
        if (n < 2) {
            return true;
        }
        for (int i = 0; i < n; i++) {
            if(arr[i] >= arr[i+1])
                return false;
        }
        return true;
    }
}
